README.md

```diff
@@ -29,6 +29,11 @@ A third party security audit was performed by Cure53, you can see the full repor
 
 `runc` only supports Linux. It must be built with Go version 1.19 or higher.
 
+NOTE there is a known issue of runc's nsenter package being incompatible
+with systems having glibc < 2.32 when compiled with Go 1.22 older than 1.22.4.
+For those systems, runc refuses to compile. The workaround is to use Go 1.21
+or Go 1.23.
```

Comment:
maybe like this?
```suggestion
, Go 1.23 or Go >= 1.22.4.
```

Reply:
So, Go 1.22.4 would work, but the issue is, I can't disable the kludge in `libcontainer/nsenter/nsenter_go122.go` for Go >= 1.22.4, there's just no way to do it. I mention this problem in the first commit as well in this PR description.

I mean, we could add some kind of a hack to Makefile which checks go version and disables this...

Reply:
OK, ended up adding a hack to Makefile to check for go1.22 version, and removed the README addition (since if runc can't be compiled it prints a self-explanatory error).

PTAL